app/src/main/java/gis2018/udacity/tametu/utils/Utils.java

```diff
@@ -26,19 +26,22 @@
      * @param context     Injected Context from caller
      * @return updated value of WorkSessionCount
      */
-    public static int updateWorkSessionCount(SharedPreferences preferences, Context context) {
+   public static int updateWorkSessionCount(SharedPreferences preferences, Context context) {
         // Retrieving value of workSessionCount (Current value of workSessionCount) from SharedPreference.
-        int oldWorkSessionCount = preferences.getInt(context.getString(R.string.work_session_count_key), 0);
+        int oldWorkSessionCount = preferences.getInt(context.getString(R.string.task_on_hand_count_key), 0);
+
 
         // Updating oldWorkSessionCount by 1.
         int newWorkSessionCount = ++oldWorkSessionCount;
```

Comment:
increment workSession and taskOnHandCount separately - both counters will hold separate values, but will be incremented at the same time.